server/server-application/src/main/java/com/saecdo18/petmily/member/service/MemberService.java

```diff
@@ -0,0 +1,29 @@
+package com.saecdo18.petmily.member.service;
+
+import com.saecdo18.petmily.member.entity.Member;
+import com.saecdo18.petmily.member.repository.MemberRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.Optional;
+
+@Transactional
+@Service
+@RequiredArgsConstructor
+public class MemberService {
+    private final MemberRepository memberRepository;
+    public long createMember(Member member){
+        Member findMember = methodVerifyNoneMember(member); //email를 통해 등록되지 않은 멤버이면 진행
+        memberRepository.save(findMember);
+        return findMember.getMemberId();
+    }
+
+    private Member methodVerifyNoneMember(Member member) {
+        Optional<Member> optionalMember = memberRepository.findByEmail(member.getEmail());
```

Comment:
아마 이렇게 쓰는게 좀더 optional 을 쓰는방법이지 않을까싶어요
```suggestion
return memberRepository.findByEmail(member.getEmail())
               orElseThrow(() -> new RuntimeException("이미 등록된 회원입니다."));
```